### PR TITLE
Removed # label from level. Close #18. Added i18n support.

### LINKED ISSUE #18: Remove usage of '#' character [1-2h]

### PR DESCRIPTION
**Pull request by protito**

### CODE REVIEW

File: Classes/LanguageManager.cpp [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+
+LanguageManager* LanguageManager::_instance = 0;
+
+LanguageManager::LanguageManager()
+{
+	string fileName;
+	// detect current language
+	switch (CCApplication::getInstance()->getCurrentLanguage())
+	{
+	case LanguageType::ENGLISH:
```
Comment (jogrimst) [inline Classes/LanguageManager.cpp]: Currently our platform only supports English, Swahili, Spanish and Arabic. So those 4 languages would be sufficient.

File: Classes/MenugScene.cpp [hunk 2]
```diff
@@ -129,7 +131,7 @@ void Menug::setupButtons()
 
 void Menug::setupVersionLabel()
 {
-    Label* label1 = Label::createWithTTF("Version 0.1", "fonts/AndikaLowerCase-Regular_5dp.ttf", 20);
+    Label* label1 = Label::createWithTTF(LanguageManager::getString("language").c_str(), "fonts/AndikaLowerCase-Regular_5dp.ttf", 10);
```
Comment (jogrimst) [inline Classes/MenugScene.cpp]: Should this be `version` instead of `language`?

Comment (protito) [inline Classes/MenugScene.cpp]: I think is better to show what language is running in the game than a version number.

File: Classes/LanguageManager.cpp (hunk 1 above)
Comment (protito) [inline Classes/LanguageManager.cpp]: The language files for the other languages will be the same as English. In the future a language can be added just by replacing the file.

File: Classes/AppDelegate.cpp [hunk 3]
```diff
@@ -15,117 +15,117 @@ AppDelegate::AppDelegate() {
...
+	// initialize director
+	auto director = Director::getInstance();
+	auto glview = director->getOpenGLView();
+
+	if (!glview) {
 #if (CC_TARGET_PLATFORM == CC_PLATFORM_MAC)
-        //glview = GLViewImpl::createWithRect("AKUA", Rect(0,0, 1024, 768));
-        glview = GLViewImpl::createWithFullScreen("AKUA");
+		//glview = GLViewImpl::createWithRect("AKUA", Rect(0,0, 1024, 768));
+		glview = GLViewImpl::createWithFullScreen("AKUA");
```
Comment (jogrimst) [inline Classes/AppDelegate.cpp]: @protito Should we replace the usage of "AKUA" here?

File: Classes/MenugScene.cpp [hunk 4]
```diff
@@ -92,55 +92,55 @@ void Menug::setupLogo()
...
-        }
-    });
-    buttonPlay->setScale(0.9f);
-    buttonPlay->setPosition(Vec2(origin.x + 3* visibleSize.width / 4, origin.y + visibleSize.height / 2 - 50));
-
-    this->addChild(buttonPlay);
+	//TODO: extract a lot of this. still doing too much here
+	float buttonX = visibleSize.width - 120;
+
+	auto buttonPlay = ui::Button::create("botonJugar.png", "botonJugarSel.png", "botonJugar.png", ui::Widget::TextureResType::PLIST);
```
Comment (jogrimst) [inline Classes/MenugScene.cpp]: Replace filename with "buttonPlay"

**Pull request merged**